crates/rust-analyzer/src/config.rs

```diff
@@ -565,6 +565,7 @@ pub struct Config {
     data: ConfigData,
     detached_files: Vec<AbsPathBuf>,
     snippets: Vec<Snippet>,
+    is_visual_studio: bool,
```

Comment:
```suggestion
    is_visual_studio_code: bool,
```
being very pedantic here, but since `visual studio` is a distinct (LSP capable) editor from `visual studio code` we should name it accordingly.